bandcamp_importer.user.js

```diff
@@ -1,17 +1,17 @@
 // ==UserScript==
 // @name           Import Bandcamp releases to MusicBrainz
 // @description    Add a button on Bandcamp's album pages to open MusicBrainz release editor with pre-filled data for the selected release
-// @version        2024.9.6.1
+// @version        2024.10.17.1
```

Comment:
```suggestion
// @version        2024.10.17.2
```
I just noticed that this version has already been released today.